release.sh

```diff
@@ -26,10 +29,21 @@ cd $TEMPDIR
 git clone git@github.com:influxdata/flux-lsp.git > /dev/null 2>&1
 cd $TEMPDIR/flux-lsp
 
-cargo bump patch --git-tag
+# Bump version
+cargo bump patch
+cargo check
+new_version=`grep "^version" Cargo.toml | awk '{print $3}' | awk -F'"' '{print $2}'`
+
+# Commit and tag release
+git add Cargo.toml
+git add Cargo.lock
+# Note: Using an annotated tag (-a) is important so that we can reliably find
+# the previous version tag.
+git tag -a -m "$new_version" "$new_version"
```

Comment:
Is there value in using the commits here, in the tag? I'm not sure; just a thought.